src/Frontend/src/views/throughputreport/setup/DiagnosticsView.vue

```diff
@@ -0,0 +1,56 @@
+<script setup lang="ts">
+import ConnectionResultView from "@/views/throughputreport/setup/ConnectionResultView.vue";
+import { useIsMonitoringEnabled } from "@/composables/serviceServiceControlUrls";
+import { useThroughputStore } from "@/stores/ThroughputStore";
+import { storeToRefs } from "pinia";
+import { ref } from "vue";
+
+const store = useThroughputStore();
+const { testResults, isBrokerTransport } = storeToRefs(store);
+const loading = ref(false);
+
+async function testConnection() {
+  loading.value = true;
+  await store.refresh();
+  loading.value = false;
+}
+</script>
+
+<template>
+  <div class="row">
+    <h6>Test Connection Results</h6>
+    <template v-if="loading">
+      <div class="sp-loader" />
+    </template>
+    <template v-else>
+      <ConnectionResultView v-if="isBrokerTransport" title="Broker" :result="testResults?.broker_connection_result!" />
+      <ConnectionResultView title="Audit" :result="testResults?.audit_connection_result!">
+        <template #instructions>
+          <a href="https://docs.particular.net/servicecontrol/servicecontrol-instances/remotes#configuration" target="_blank">Learn how to configure audit instances</a>
+        </template>
+      </ConnectionResultView>
+      <ConnectionResultView v-if="useIsMonitoringEnabled()" title="Monitoring" :result="testResults?.monitoring_connection_result!">
+        <template #instructions>
+          <a href="https://docs.particular.net/servicecontrol/monitoring-instances/installation/creating-config-file" target="_blank">Learn how to configure monitor instances</a>
```

Comment:
check: are there any links in servicepulse that open in a new tab that (correctly) have the icon next to them indicating that they will open in a new tab? This should be used here, at the end of the text https://www.iconfinder.com/icons/192090/external_go_to_hyperlink_link_new_tab_outside_url_icon#size=128